src/Canvas.js

```diff
@@ -4,6 +4,7 @@
  */
 "use strict";
 
+var _               = require('lodash');
```

Comment:
I would prefer not to pull in the entire Lodash library here just for a few utility methods. Would be better to either just require the functions we need or use something like https://github.com/megawac/babel-plugin-lodash and using es6 imports